### DISCUSSION TITLE
Silo license

### DISCUSSION THREAD

**Discussion by esys-escript** [LLNL/Silo#301] [Q&A] [answered]
Could you please tell me what license Silo uses?

**Comment by markcmiller86** [comment, marked as answer]
Sorry for confusion. Silo is BSD 3-clause license. You can [read more about it](https://github.com/LLNL/Silo/blob/main/BSD_LICENSE_README) in the `BSD_LICENSE_README` file at top level.

Silo was *originally* released in early 2000's using LLNL-home-grown license verbiage. It was re-released as BSD 3-clause. However, two built-in compression libs ([fpzip](https://github.com/LLNL/fpzip) and [hzip](https://computing.llnl.gov/projects/hzip)) DID NOT FIT under the BSD release. So, we provided [kinds of download files](https://github.com/LLNL/Silo/releases). Those with `bsd` in the file names are BSD licensed code only. Those without are released under Silo's *original* license.

Both forms of license are explained in `BSD_LICENSE_README`.

We will get this properly hooked up to GitHub's license buttons, etc. in the upcoming release.
